lib/cn/csched_sp3.c

```diff
@@ -868,7 +868,7 @@ sp3_dirty_node_locked(struct sp3 *sp, struct cn_tree_node *tn)
             /* Leaf nodes sorted by vgroup scatter and garbage.
              */
             if (scatter > 0) {
-                weight = ((uint64_t)scatter << 32) | garbage;
+                weight = ((uint64_t)scatter << 32) | (UINT32_MAX - garbage);
```

Comment:
For scatter remediation, prefer nodes with less garbage over those with more garbage to allow the GC rule an oppotunity to tackle the nodes with high garbage.  I actually think this should computed differently, something like ```weight = scatter * (UINT32_MAX - garbage)``` or some such, need to revisit when we rethink scatter remediation.